static/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -95,16 +94,6 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
     return this.props.eventSlug.split(':')[0];
   }
 
-  generateTagKey = (tag: EventTag) => {
```

Comment:
Does this break custom tags with the same name as the reserved tag names? This was "fixed" in https://github.com/getsentry/sentry/pull/19571

Reply:
Hm... good point. Perhaps the right thing to do here is to keep the `tags[...]` syntax and add it to the other pages as well 🤔